src/main/java/com/openshift/restclient/capability/resources/IBuildTriggerable.java

```diff
@@ -30,7 +32,34 @@
 	 * Trigger a build with the given source level commit id
 	 * @param commitId
 	 * @return The build that was triggered
+	 * @deprecated
 	 */
 	IBuild trigger(String commitId);
+	
+	/**
+	 * Set the commit level for the git clone extraction
```

Comment:
A nit, but I wonder if we should refrain from mentioning 'Git' since in theory we could support an alternate source control mechanism